vagrant/provisioning/roles/minio/tasks/main.yml

```diff
@@ -0,0 +1,52 @@
+
+- name: create minio group
+  become: yes
+  group:
+    name: minio
+    gid: 9002
+    state: present
+- name: create minio user
+  become: yes
+  user:
+    name: minio
+    group: minio
+    shell: /sbin/nologin
+    home: /opt/minio
+- name: create minio directories
+  become: yes
+  file:
+    path: /opt/minio/{{item}}
+    state: directory
+    owner: minio
+  with_items:
+    - 'bin'
+    - 'data'
+- name: download minio file from internet
+  become: yes
+  get_url:
+    url: "https://dl.minio.io/server/minio/release/linux-amd64/minio"
+    dest: /opt/minio/bin/
+    mode: '0777'
+- name: create minio config file
+  become: yes
+  copy:
+    dest: /opt/minio/minio.conf
```

Comment:
Let's use {{ root_folder }} variable here so instead /opt/minio/minio.conf we should use {{ root_folder }}/app/minio/minio.conf

We use {{ root_folder }}/app for application location and {{ root_folder }}/data for data the service is storing and {{ root_folder }}/log for the log location

Reply:
{{ root_folder }}/tmp for temp files, if MinIO uses them, and {{ root_folder }}/install to download install artifacts into...